client/client.go

```diff
@@ -60,11 +58,6 @@ func NewClient(endpoint string) (Client, error) {
 		return nil, err
 	}
 
-	evmlogs, err := NewEvmLogsClient(endpointURL)
-	if err != nil {
-		return nil, err
-	}
-
 	contract, err := NewContractClient(endpointURL)
```

Comment:
Is there a reason we can't pass the client in here?

Reply:
fixed in 14bdc3d